### ISSUE TITLE
GitHub wiki

### ISSUE THREAD

**Issue by thejohnbrown** [blacklanternsecurity/writehat#98]
Can the wiki for this repository be turned on so we can start creating/collecting user guides for the various features in writehat? I know we have the readme already but I feel that will become inefficient to search through when it becomes too big.

**Comment by unixerius** [comment]
I agree. It would be great if the community can start contributing to documentation.
